lib/containers/utils.pm

```diff
@@ -41,52 +41,69 @@ sub test_seccomp {
     }
 }
 
+sub setup_registry {
+    my $runtime  = shift;
+    my $registry = get_var('REGISTRY');
+    if ($registry) {
+        assert_script_run('openssl s_client -servername ' . $registry . ' -connect ' . $registry . ':443 < /dev/null 2>/dev/null | openssl x509 > /etc/pki/trust/anchors/certificate.crt');
+        assert_script_run('update-ca-certificates');
+        systemctl('restart docker') if ($runtime eq 'docker');
+        record_info('Registry', "Available images in $registry:\n" . script_output("curl https://$registry/v2/_catalog"));
+    }
+}
+
 sub basic_container_tests {
     my $runtime = shift;
     die "You must define the runtime!" unless $runtime;
+    my $registry             = get_var('REGISTRY', 'docker.io');
+    my $alpine_image_version = '3.6';
```

Comment:
I guess, this will change in future, maybe it is worth to set the version as openQA variable. WDYT?

Reply:
IMO no, as arguably the test code knows best which version it was written for.

Reply:
yes, why not?

Reply:
Just to be clear, I am fine with both. :) It looks to me more handy to update the version in openQA directly rather than creating PR to make a version bump.

Reply:
It's easier, but that also means it has to be done everywhere separately (and a fallback value defined in the test code anyway) and there's no verification run for possibly breaking version bumps.

Reply:
> IMO no, as arguably the test code knows best which version it was written for.

Sorry, didn't read your comment before and I amended. Anyway, the code now uses 3.6 by default, so you can omit that variable anyway.

Reply:
> It's easier, but that also means it has to be done everywhere separately (and a fallback value defined in the test code anyway) and there's no verification run for possibly breaking version bumps.

Yes, it would need to be in the openQA's test suite definition table. Instead of fallback value, I would personally prefer to die (`get_required_var`).

Reply:
If we create variable for it, anyone can change the version in the UI, but that can have a side effect that the image version is not in the remote `REGISTRY` and it will fail. If we hardcode it, we force having a PR where we make sure we also add the version in the remote `REGISTRY`. 
I like the idea of having a variable for it but it can easily break things if the tester is now aware of this dependency. 
BTW, the same would apply for `python:3` image...